internal/event/target/postgresql.go

```diff
@@ -444,3 +449,15 @@ func NewPostgreSQLTarget(id string, args PostgreSQLArgs, loggerOnce logger.LogOn
 
 	return target, nil
 }
+
+var errInvalidPsqlTablename = errors.New("invalid PostgreSQL table")
+
+func validatePsqlTableName(name string) error {
+	// check for simple name or quoted name
+	if match, err := regexp.MatchString("^(([a-zA-Z_][a-zA-Z0-9_$]*)|(\"[^\"]+\"))$", name); err != nil {
```

Comment:
I have only a vague idea what the regex does. Could you add a comment with the intent?

Reply:
It's in the line above :-) It simply checks for t